crates/bevy_ecs/src/schedule/schedule.rs

```diff
@@ -760,6 +760,27 @@ enum UninitializedId {
     },
 }
 
+/// Metadata for system sets in a schedule.
+#[derive(Default)]
+struct SystemSets {
+    /// List of system sets in the schedule
+    system_sets: SlotMap<SystemSetKey, SystemSetNode>,
```

Comment:
I would rename these to reduce the stutter. 
```suggestion
    sets: SlotMap<SystemSetKey, SystemSetNode>,
```

Reply:
Oh, that is much nicer!  It also restores a few places where the lines got too long and `rustfmt` spilled them.  Thanks!